canutils/lely-canopen/Makefile

```diff
@@ -170,25 +170,34 @@ MAINSRC    = $(LELYCANOPEN_SRCDIR)/tools/coctl.c
 endif
 
 # Download and unpack tarball if no git repo found
-ifeq ($(wildcard $(LELYCANOPEN_SRCNAME)/.git),)
+ifeq ($(wildcard $(LELYCANOPEN_SRCNAME)),)
 $(LELYCANOPEN_TARBALL):
 	@echo "Downloading: $(LELYCANOPEN_TARBALL)"
 	$(Q) curl -L -O $(CONFIG_CANUTILS_LELYCANOPEN_URL)/$(LELYCANOPEN_TARBALL)
 
 $(LELYCANOPEN_SRCNAME): $(LELYCANOPEN_TARBALL)
 	@echo "Unpacking: $(LELYCANOPEN_TARBALL) -> $(LELYCANOPEN_UNPACKNAME)"
 	$(Q) $(UNPACK) $(LELYCANOPEN_TARBALL)
+
+patch_src: $(LELYCANOPEN_SRCNAME)
 	# Get the name of the directory created by the tar command
 	$(eval LELYCANOPEN_UNPACKNAME := $(shell ls -d lely-core-master*))
 	$(Q) mv $(LELYCANOPEN_UNPACKNAME) $(LELYCANOPEN_SRCNAME)
-	$(Q) cat 0001-NuttX-port.patch | patch -s -N -d $(LELYCANOPEN_SRCNAME) -p1
+	$(Q) cat 0001-tools-eliminate-multiple-definitions-of-poll-compile.patch | patch -s -N -d $(LELYCANOPEN_SRCNAME) -p1
+	$(Q) cat 0002-tools-coctl.c-use-readline-and-fix-printf-issues.patch | patch -s -N -d $(LELYCANOPEN_SRCNAME) -p1
+	$(Q) cat 0003-src-co-nmt.c-fix-compilation.patch | patch -s -N -d $(LELYCANOPEN_SRCNAME) -p1
+	$(Q) cat 0004-tools-coctl.c-add-missing-mutex-init.patch | patch -s -N -d $(LELYCANOPEN_SRCNAME) -p1
+	$(Q) cat 0005-add-NuttX-support.patch | patch -s -N -d $(LELYCANOPEN_SRCNAME) -p1
 	$(Q) echo "Patching $(LELYCANOPEN_SRCNAME)"
-endif
 
+context:: patch_src
+else
 context:: $(LELYCANOPEN_SRCNAME)
+endif
+
 
 distclean::
-ifeq ($(wildcard $(LELYCANOPEN_SRCNAME)/.git),)
```

Comment:
why change? we need skip remove the directory which contian git history

Reply:
for some reason this stopped work correctly. But the real problem was probably in `eval LELYCANOPEN_UNPACKNAME` and parallel execution because now I see it works as expected.
I restored changes realted to `.git`